ivy/stateful/module.py

```diff
@@ -144,6 +145,9 @@ def __init__(
         self._target = None
         self._lazy_compiled = False
         self._dynamic_backend = dynamic_backend
+        if hasattr(self, "_create_buffers"):
```

Comment:
Seems like we're still using `_create_buffers` and `_buffer_tracker`, could you please remove it as we'd only need to allow users to register buffers "on-spot"?